target/linux/ramips/image/mt76x8.mk

```diff
@@ -156,6 +156,13 @@ define Device/pbr-d1
 endef
 TARGET_DEVICES += pbr-d1
 
+define Device/rakwireless_rak633
+  DTS := RAK633
+  DEVICE_TITLE := RAKwireless RAK633
```

Comment:
> Shenzhen Rakwireless Technology Co., Ltd. (RAK) was established in June 2014. 

@eikepedia So it should be probably written as `Rakwireless` and not RAKwireless.